bugbot/rules/has_str_no_range.py

```diff
@@ -109,7 +109,7 @@ def get_bz_params(self, date):
             "n6": 1,
             "f6": "reporter",
             "o6": "regexp",
-            "v6": r"^.*@softvision\.(ro|com)$",
```

Comment:
Why are we not including Softvision email anymore?

Reply:
QA is not using those emails anymore